src/website/views/components/brand.pug

```diff
@@ -30,25 +30,70 @@ h4 Base
+        <svg xmlns="http://www.w3.org/2000/svg" id="centurylink-logo" viewBox="0 0 204 41" version="1.1">
+          <g id="logo">
```

Comment:
This SVG code looks different from the base. They should all be consistent, only difference will be color values. Code should be minified too.